templates/cpan.conf.erb

```diff
@@ -15,14 +15,14 @@ $CPAN::Config = {
   'curl' => q[/usr/bin/curl],
   'ftp' => q[/usr/bin/ftp],
   'ftp_passive' => q[1],
-  'ftp_proxy' => q[],
+  'ftp_proxy' => q[<%=@ftp_proxy%>],
```

Comment:
please check ``@ftp_proxy`` is not undefined before adding this line in the config